docs/fptf.proto

```diff
@@ -51,15 +51,15 @@ message Route {
 }
 
 message Schedule {
-    message SequencePart {
-        optional int32 arrival = 1 // sometimes actually required
-        optional int32 departure = 2  // sometimes actually required
+    message Element {
```

Comment:
Why `Element`? imho it is not intuitive.

Reply:
See @kiliankoe s comment above 😄